### ISSUE TITLE
Post a version for RVHS

### ISSUE THREAD

**Issue by samreid** [phetsims/protein-synthesis#29]
For unknown reasons, bayes is failing today, and its data directory seems corrupted:

![image](https://user-images.githubusercontent.com/679486/68312731-bb7ba080-0070-11ea-8ed7-03bdd7bf74e9.png)

I tried posting phetsims.github.io, but that was blocked by the school firewall.  Instead I placed a version at https://phet.colorado.edu/files/protein-synthesis.html

This file will be used in the classroom for the next few business days, but then it can be cleaned up.  Self-assigning and putting on hold for cleanup steps.

**Comment by samreid** [comment]
I'm inclined to leave this version there in case it is needed again, closing.
